packages/remirror__react-components/src/floating-menu.tsx

```diff
@@ -33,13 +40,6 @@ interface BaseFloatingPositioner extends UseEditorFocusProps {
    */
   hideWhenInvisible?: boolean;
 
-  /**
-   * Set animated as detailed [here](https://reakit.io/docs/popover/#animating).
-   *
-   * Currently this is turned off due to problems with an infinite loop.
-   */
-  animated?: boolean | number;
-
```

Comment:
I don't think we should remove any option in a non-major version, unless this option is never been used at all.

Reply:
okay.I'll remove the commit.